status/reporter_gitlab_test.go

```diff
@@ -187,7 +188,7 @@ var _ = Describe("GitLabReporter", func() {
 		},
 			Entry("Missing repo_url", gitops.PipelineAsCodeRepoURLAnnotation, false),
 			Entry("Missing SHA", gitops.PipelineAsCodeSHALabel, true),
-			Entry("Missing project ID", gitops.PipelineAsCodeSourceProjectIDAnnotation, false),
```

Comment:
This should be addition to test cases, not a replacement

Reply:
Fixed. Thanks.